components/main-chef-wrapper/main.go

```diff
@@ -106,6 +113,15 @@ Available Commands:
 	fmt.Printf(msg)
 }
 
+func doStartupTasks() error {
+	dir, err := homedir.Expand("~/.chef")
+	if err != nil {
+		return err
+	}
+	err = os.MkdirAll(dir, 0700)
```

Comment:
In some weird case where they don't have a home directory and we create it for them, do you think there are any issues with `700` permissions on that directory? I would think that would be desired but my default on Darwin is `755` (which I am sure is because of my umask).

Just want to make sure we are intentional about this setting.

Reply:
I'll play it safe and skip creation if we can't find an existing valid home directory.

Reply:
Final version - because it's safe for us to continue running even if we don't make the directory, I updated this to just try to make the dir (using Mkdir, not MkdirAll) and move on.